_includes/footer.html

```diff
@@ -14,7 +14,8 @@
 </section>
 <div class="footer">
   <div class="container">
-    <p>This web site is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_GB">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.  Source available: <a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/theforeman/theforeman.org" rel="dct:source">github/theforeman/theforeman.org</a></p>
+    <p>Edit and improve <a href="{{ site.github.repository_url }}/edit/gh-pages/{{ page.path }}">this page on GitHub</a>. All sources available: <a xmlns:dct="http://purl.org/dc/terms/" href="{{ site.github.repository_url }}" rel="dct:source">github/{{ site.repository }}</a></p>
```

Comment:
This is neat, but would it be better in a more prominent place? I don't think many people would notice this and it might encourage more contributions to pages like the manual.

Maybe something like `[ Edit ]` right aligned under the header?